{"name": "Ручка /api/v1/orders/accept/", "status": "passed", "description": "Создаём заказ, получаем трек номер, принимаем заказ с некорректным id курьера", "start": 1710617468373, "stop": 1710617468428, "uuid": "8336966e-ca1e-4aa1-b8e8-8582af035f13", "historyId": "a880846fcbfb59b7d59484d90ca1336b", "testCaseId": "a880846fcbfb59b7d59484d90ca1336b", "fullName": "test_api.tests.test_accept_order.TestAcceptOrder#test_accept_order_wrong_courier_id_error", "labels": [{"name": "parentSuite", "value": "test_api.tests"}, {"name": "suite", "value": "test_accept_order"}, {"name": "subSuite", "value": "TestAcceptOrder"}, {"name": "host", "value": "LAPTOP-KS9V29SF"}, {"name": "thread", "value": "5592-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_api.tests.test_accept_order"}]}